### PR TITLE
add parameters_mapping schema role

### PR DESCRIPTION
**Pull request by evacougnon**

### CODE REVIEW

File: private-sample/data_bags/postgresql_roles/db_po_main.json [hunk 1]
```diff
@@ -763,6 +763,12 @@
         {
             "name": "plankton_products",
             "password": "plankton_products"
+        },
+        {
+            "name": "parameters_mapping",
+            "password": "parameters_mapping"
+	    "memberof": [
```
Comment (lbesnard) [inline private-sample/data_bags/postgresql_roles/db_po_main.json:770]: fix indent

File: private-sample/data_bags/postgresql_roles/db_po_main.json [hunk 2]
```diff
@@ -763,6 +763,12 @@
         {
             "name": "plankton_products",
             "password": "plankton_products"
+        },
+        {
+            "name": "parameters_mapping",
+            "password": "parameters_mapping"
```
Comment (lbesnard) [inline private-sample/data_bags/postgresql_roles/db_po_main.json:769]: no comma at the end of the line

**Comment by evacougnon** [comment]
@lbesnard , it should look better now!

**Comment by lbesnard** [comment]
you must have pushed-force as I was looking at it as there was a missing ```]```

**Pull request merged**